lightning/src/ln/functional_test_utils.rs

```diff
@@ -1780,7 +1780,7 @@ macro_rules! expect_payment_forwarded {
 		let events = $node.node.get_and_clear_pending_events();
 		assert_eq!(events.len(), 1);
 		match events[0] {
-			Event::PaymentForwarded { fee_earned_msat, prev_channel_id, claim_from_onchain_tx, next_channel_id } => {
+			Event::PaymentForwarded { fee_earned_msat, prev_channel_id, claim_from_onchain_tx, next_channel_id, .. } => {
```

Comment:
Question: for this test macro, if I add the new field, I get stuck trying to update the `mid_update_fulfill_dance` macro, because I can't see how to get the amount. I notice that fee is obtained there in a perhaps roundabout way, by getting the `forwarding_fee_base_msat` from the channel config (and using no proportional fee), which suggests getting the amount from the HTLC might not straightforward here?

Reply:
Then if everyone's okay with it I'll go with the "just not worry about it" option